WooCommerce/WooCommerceTests/POS/ViewModels/TotalsViewModelTests.swift

```diff
@@ -22,8 +24,37 @@ final class TotalsViewModelTests: XCTestCase {
                               currencyFormatter: .init(currencySettings: .init()),
                               paymentState: .acceptingCard,
                               isSyncingOrder: false)
+        cancellables = []
+    }
+    override func tearDown() {
+        cancellables.forEach {
+            $0.cancel()
+        }
+        cancellables.removeAll()
+
+        super.tearDown()
+    }
+    func test_isSyncingOrder() async throws {
```

Comment:
Let's use the `test_X_when_Y_then_Z` test name descriptions (when possible) for consistency with the rest of test suite and to be more descriptive of what the test does.